modules/nw-ingress-configuring-application-domain.adoc

```diff
@@ -44,7 +44,7 @@ spec:
   appsDomain: <test.example.com>      <2>
 ----
 <1> Default domain
-<2> Optional: Domain for AWS infrastructure to use for application routes. Instead of the default prefix, `apps`, you can use an alternative prefix like `test`.
+<2> Optional: Domain for cloud platform infrastructure to use for application routes. Instead of the default prefix, `apps`, you can use an alternative prefix like `test`.
```

Comment:
```suggestion
<2> Optional: Domain for {product-title} to use for application routes. Instead of the default prefix, `apps`, you can use an alternative prefix like `test`.
```